src/components/header.tsx

```diff
@@ -13,8 +14,10 @@ import {
   DropdownMenuLabel,
   DropdownMenuSeparator,
 } from "components/ui/dropdown-menu";
+import NewUserButton from "./user-button";
 
-export default function Header() {
+// Alert component on header
+function Alert({ isLoggedIn }: { isLoggedIn: boolean | undefined }) {
```

Comment:
we usually want to have one component per file, so could you move this to a new file? maybe `alert-dropdown.tsx`